src/CentreonRemote/Application/Clapi/CentreonRemoteServer.php

```diff
@@ -31,131 +31,7 @@ public static function getName() : string
         return (new \ReflectionClass(__CLASS__))->getShortName();
     }
 
-    /**
-     * Clapi command to enable remote server
-     *
-     * @param string $parametersString parameters string made of
-     * a comma separated list of hosts/urls/ip adresses representing the central,
-     * a boolean to enable/disable certificate check to contact the central,
-     * the method to use to contact the remote (http or https),
-     * the http port to use to contact the remote,
-     * a boolean to enable/disable certificate check to contact the remote,
-     * a boolean to enable/disable the use of proxy to contact the central
-     * @return int|void
-     */
-    public function enableRemote(string $parametersString)
```

Comment:
Should be discuss with PM before removing this method

Reply:
ok!